chart/templates/deployment.yaml

```diff
@@ -46,6 +52,11 @@ spec:
             httpGet:
               path: /
               port: http
+              {{- if .Values.basic_auth.enabled }}
```

Comment:
@bykvaadm 

Use the following instead:
````
  {{- if .Values.env.BASIC_AUTH_USERNAME }}
  
  {{- end }}
````

in both probes and remove the if-block above and the additions in values.yaml.

Reply:
That way it remains KISS :)

Reply:
the if-block above must remain so user can disable probes if they wish

Reply:
if-block above =
````
            {{- if .Values.basic_auth.enabled }}
            - name: BASIC_AUTH_USERNAME
              value: "{{ .Values.basic_auth.username }}"
            - name: BASIC_AUTH_PASSWORD
              value: "{{ .Values.basic_auth.password }}"
            {{- end }}
````